test/e2e/e2e_suite_test.go

```diff
@@ -335,6 +335,8 @@ var _ = SynchronizedAfterSuite(func() {
 func initScheme() *runtime.Scheme {
 	sc := runtime.NewScheme()
 	framework.TryAddDefaultSchemes(sc)
+	// TODO: should probably be added to TryAddDefaultSchemes in core CAPI.
```

Comment:
Can we implement this TODO?

Reply:
I can implement it, but not sure if it's worth waiting an entire bump-cycle just to avoid one line here

Reply:
Let's see, maybe I'll pick it up if Fabrizio's PR merges before mine: https://github.com/kubernetes-sigs/cluster-api/pull/10745/files